src/app/core/services/profile/profile.service.ts

```diff
@@ -171,4 +173,24 @@ export class ProfileService {
       }
     }).catch(error => {})
   }
+
+  async prefillData(requestDetails: any,entityNames:any,formData:any,showAddOption:any=true) {
+    let existingData = requestDetails;
+    if(requestDetails?.about){
+       existingData = await this.form.formatEntityOptions(requestDetails,entityNames)
+    }
+    for (let i = 0; i < formData.controls.length; i++) {
+      if(formData.controls[i].type == 'chip'){
+        formData.controls[i].meta.showAddOption = showAddOption;
+      }
+      formData.controls[i].value = existingData[formData.controls[i].name] ? existingData[formData.controls[i].name] : '';
+      formData.controls[i].options = _.unionBy(
+        formData.controls[i].options,
+        formData.controls[i].value,
+        'value'
```

Comment:
Also nothing is getting returned after prefill

Reply:
no return required here form will update directly from this page